### PR TITLE
fix: Avatar iv null bug

### PR DESCRIPTION
**Pull request by peterstarling**
https://trello.com/c/FBMD06GE/1104-team-lee-is-breaking-network

### CODE REVIEW

**Comment by kmaid** [comment]
Maybe this could use a test?

**Comment by johnytiago** [comment]
It does @kmaid. I'm on it! 😉

**Comment by kmaid** [comment]
Are responses from Squidex typed? That probably needs fixed too

**Comment by peterstarling** [comment]
i thought we were monkey patching to get that fix out of the way. Proper fix requires not just a test but also changes to our typings which will cascade a series of type errors that need to be addressed

**Comment by peterstarling** [comment]
et voila, here is the test

File: apps/asap-server/src/controllers/teams.ts [hunk 1]
```diff
@@ -94,7 +94,11 @@ const transformRestTeamMember = (
     role: get(user, 'data.teams.iv', []).find(
       (t: { id: string[] }) => t.id[0] === teamId,
     ).role,
-    avatarUrl: user.data.avatar && createURL(user.data.avatar.iv)[0],
+    avatarUrl:
+      (user.data.avatar &&
+        user.data.avatar.iv &&
+        createURL(user.data.avatar.iv)[0]) ||
+      undefined,
```
Comment (johnytiago) [inline apps/asap-server/src/controllers/teams.ts:101]:
```suggestion
      user.data.avatar?.iv ?  createURL(user.data.avatar.iv)[0]) : undefined,
```

**Review by johnytiago — APPROVED** [review]

**Pull request merged**